test/unit/src/common/components/repository-input/t_repository-input.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import expect from 'expect';
 import { shallow } from 'enzyme';
-import configureStore from 'redux-mock-store';
-
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
 // Workaround for Redux-wrapped components
 // See: https://github.com/airbnb/enzyme/issues/472
 import Component from '../../../../../../src/common/components/repository-input';
 const RepositoryInput = Component.WrappedComponent;
 
-let dispatch = expect.createSpy();
+//let dispatch = expect.createSpy();
```

Comment:
Good catch.

`setGitHubRepository` that used to just return action object now returns function that dispatches 2 other actions... But it may all be irrelevant now that webhook errors are displayed on `/setup` view rather than in repository input.
https://github.com/canonical-ols/build.snapcraft.io/pull/65/files#diff-890863cbc28566af9b5f84d509b1d749R19